Documentation/aws.md

```diff
@@ -0,0 +1,28 @@
+---
+title: Running Rook in AWS
+weight: 25
+indent: true
+---
+
+# Running Rook in AWS
+
+Rook can functionally replace Elastic Block Storage (EBS), Elastic File System (EFS) and S3 with Rook ObjectStore. Potentially reducing storage cost, increasing performance and portability of your Kubernetes workloads.
+
+:information_source:  
+Please refer to [Creating Rook Clusters][3] on how to customize your cluster to use specific devices.
+
+* To deliver robust performance at reasonable cost, consider using instances backed by [*instance store*][1] for Rook cluster nodes. Utilization of storage devices directly attached to the instance will reduce latency and provide more consistent IO performance.
+* The ephemeral nature of the *instance store* is handled by resiliency build into Rook. Make sure to configure `replicas` parameter in accordance with your durability requirements.
```

Comment:
what about this wording? 
"Durability of your data when consuming the *instance store* is handled by Rook. You can configure the durability in Rook by setting the `replicas` parameter on the data pools in accordance with your requirements."